hash/hasher_bcrypt.go

```diff
@@ -45,6 +45,11 @@ func (h *Bcrypt) Generate(ctx context.Context, password []byte) ([]byte, error)
 		return nil, err
 	}
 
+	// ensure that the context is not canceled before doing the heavy lifting
+	if ctx.Err() != nil {
```

Comment:
This is the relevant context check for the import API.

Reply:
Ok, I see, it is called in a loop and if we don't check the context then the loop just continues. Got it!